nxviz/plots.py

```diff
@@ -80,6 +80,12 @@ class BasePlot(object):
 
     :param edgeprops: A `matplotlib-compatible `props` dictioanry.
     :type edgeprops: `dict`
+
+    :param fontsize: A text property for editing size of labels in graph
+    :type fontsize: int
+
+    :param fontfamily: A text property to define font family
```

Comment:
Referring the comment below:

```suggestion
    :param fontfamily: A text property to define font family. Valid inputs ('serif', 'sans-serif', 'fantasy', 'monospace').
```